### PR TITLE
Update inner-product-argument.md

### PR DESCRIPTION
**Pull request by marcobesier**
Only minor adjustments, as usual. However, I've noticed that the bullet point list in the Setup subsection is not rendered properly on the website. When I view it on GitHub, though, everything's rendered correctly. Given that I can't troubleshoot this myself, I left it untouched. It might be related to the fact that the bullets use math mode. Maybe inserting one or more extra "{ } wrappers" might do the trick...

### CODE REVIEW

File: content/inner-product-argument/en/inner-product-argument.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ $$\mathbf{l}(2)\circ\mathbf{r}(2) = \begin{bmatrix}5\\8\end{bmatrix}\circ\begin{
 In other words, multiplying two vector polynomials together and then evaluating the product at some point is the same as evaluating the vector polynomials separately and then taking the Hadamard product on the resulting vectors.
 
 ## Inner product of vector polynomials
-To compute the inner product of two vector polynomials, we multiply them together as described above, but then sum up all the vectors so they become scalars. We denote this operation as $\langle \mathbf{l}(x), \mathbf{r}(x) \rangle$. We can accomplish the same thing by using the inner product when we multiply vector coefficients instead of using the Hadamard product.
+To compute the inner product of two vector polynomials, we multiply them together as described above, but then sum up the vector coordinates so the result becomes a scalar. We denote this operation as $\langle \mathbf{l}(x), \mathbf{r}(x) \rangle$. We can accomplish the same thing by using the inner product when we multiply vector coefficients instead of using the Hadamard product.
```
Comment (RareSkills) [inline content/inner-product-argument/en/inner-product-argument.md]: "vector coordinates" doesn't feel like a conventional way to say it. Maybe "vector entries"?

**Review by RareSkills — CHANGES_REQUESTED** [review]
smol change requests

File: content/inner-product-argument/en/inner-product-argument.md (hunk 1 above)
Comment (marcobesier) [reply content/inner-product-argument/en/inner-product-argument.md]: Fair enough. Was thinking about calling it entries as well. Will change that. 👍

**Comment by marcobesier** [comment]
Done 🤝

**Review by RareSkills — APPROVED** [review]

**Pull request merged**